splice_si_api/src/main/java/com/splicemachine/si/impl/store/ActiveIgnoreTxnCacheSupplier.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2012 - 2017 Splice Machine, Inc.
+ *
+ * This file is part of Splice Machine.
+ * Splice Machine is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3, or (at your option) any later version.
+ * Splice Machine is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License along with Splice Machine.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.splicemachine.si.impl.store;
+
+import com.splicemachine.access.api.PartitionAdmin;
+import com.splicemachine.access.configuration.HBaseConfiguration;
+import com.splicemachine.utils.SpliceLogUtils;
+import org.apache.hadoop.hbase.util.Pair;
+import org.apache.log4j.Logger;
+
+import java.util.*;
+
+/**
+ * Created by jyuan on 10/30/17.
+ */
+public class ActiveIgnoreTxnCacheSupplier {
```

Comment:
I think the name of this class is confusing, I guess you made it symmetrical to the ActiveTxnCacheSupplier, but in this case it doesn't deal with Active transactions so I'd rename it.